Updated airgap docs to specify dependencies required for Selinux, Cent/Rhel 8 (#2385)
rancher/rke2: docs/install/airgap.md

All files mentioned in the steps can be obtained from the assets of the desired released rke2 version [here](https://github.com/rancher/rke2/releases).

> If running on a SELinux system enforcing air-gapped node, you must first install the necessary SELinux policy RPM before performing these steps. See our [RPM Documentation](https://docs.rke2.io/install/methods/#rpm) to determine what you need.

Reviewer: ```suggestion
If running on an air-gapped node with SELinux enabled, you must manually install the necessary SELinux policy RPM before performing these steps. See our [RPM Documentation](https://docs.rke2.io/install/methods/#rpm) to determine what you need.
```

> If running on a SELinux, Centos, or RHEL 8 system enforcing air-gapped node, the following are required dependencies when doing an [RPM install](https://docs.rke2.io/install/methods/#rpm):

Reviewer 2: Nit: 

```suggestion
If running on a SELinux, CentOS, or RHEL 8 system enforcing air-gapped node, the following are required dependencies when doing an [RPM install](https://docs.rke2.io/install/methods/#rpm):
```

> If running on a SELinux, CentOS, or RHEL 8 system enforcing air-gapped node, the following are required dependencies when doing an [RPM install](https://docs.rke2.io/install/methods/#rpm):

Reviewer: ```suggestion
If running on an air-gapped node running SELinux, CentOS, or RHEL 8 with SELinux enabled, the following are required dependencies when doing an [RPM install](https://docs.rke2.io/install/methods/#rpm):
```
